### ISSUE TITLE
For Sprint 3

### ISSUE THREAD

**Issue by SaketPenurkar** [cu-ecen-5013/final-project-SaketPenurkar#4]
1) Development of MQTT application code to accept values of both the temperature and humidity sensor values

- [x] DoD: Developing a python script to accept the values of the sensors over MQTT and keep on saving the values to the text file acting as a database.

Commit Link - [Usr folder has python codes for accepting sensor values](https://github.com/cu-ecen-5013/final-project-SaketPenurkar/tree/e8f10d0de96b2d174fcdaa303e7d7bbfde10bd2b)

2) Finishing up the remaining part of the Web page to pull up temperature and humidity data from the database

- [x] DoD: Creating the link between the html and the file database to display sensor data on Webpage

Commit Link - [Interface of HTML page with file](https://github.com/cu-ecen-5013/final-project-SaketPenurkar/tree/e8f10d0de96b2d174fcdaa303e7d7bbfde10bd2b)

3) Boot up functionality

- [x] DoD: Adding the functionality for the python script to run on boot up. The script will start accepting the connections from the clients and keep on writing the values on the data base.
Commit Link - [Boot up script for server](https://github.com/cu-ecen-5013/final-project-SaketPenurkar/tree/4f82e28ccd084339d6fc38c0af792b74fc1edc2b)

**Comment by SaketPenurkar** [comment]
Blocker 1; The PHP package has been linked with the HTML web page on virtual machine for testing purposes. The same has to be tested on Raspberry pi platform

**Comment by SaketPenurkar** [comment]
For DoD 2 -> 

The image below shows the web page of the server at address of '10.0.0.20' in local network being accessed remotely.
The temperature data and the humidity data from the ICU can be seen differently.
The Web page accesses a file, which acts as a database. The temperature and humidity values are shown on the web page as they are accessed from the file.
 
![Capture](https://user-images.githubusercontent.com/54698977/80334385-4bbfc480-880e-11ea-9d20-3e87a9f313b8.PNG)

**Comment by SaketPenurkar** [comment]
**With reference to the above blocker -**
The original idea was to integrate PHP with the SQL database on Raspberry Pi. However, I switched to using a text file as a database and fetching the sensor values on the Web page.

Tried the HTML + PHP work on Virtual machine, which was successful.
Commit Link - [HTML + PHP testing on virtual machine ](https://github.com/cu-ecen-5013/final-project-SaketPenurkar/tree/3046d0a2ec53def9827fac5c9736835aa27e2a9f)

The blocker is removed as I replicated the text file as database using buildroot on Raspberry Pi and the functionality is working.
